code/modules/surgery/ipc_revival.dm

```diff
@@ -88,3 +88,17 @@
 	-After reactivation, use a screwdriver to screw the panel back into place.<BR>\
 	-If the unit breaks down shortly after reactivation, replace any missing internal components and reboot again if necessary.<BR>\
 	Nanotrasen is not liable for any damages caused during the repair process."
+
+/datum/surgery/synth_revival
+	name = "Synthetic Reactivation"
```

Comment:
Sbould probably make sure this only shows up on synths, rather than everyone with a robotic head